index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
```

Comment:
You could move this span inside of the `<h1>` along with the other text.  That way the page title will read "A Viget Exploration - Improving Medical Adherence" (although you may need to add " - " to your `<img>`'s alt text for it to read that way.